src/recipes/irish-winter.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "Irish Winter Spiked Coffee",
+    "description": "The Irish winter recipe offers a wonderful way to spike your coffee. It's a subtle transformation of the classic Irish coffee and is so easy to make that you'll be mixing it up regularly.",
+    "github": "divyhshah",
+    "ingredients": [
+        {
+            "quantity": "1 1/2",
+            "measure": "ounces",
+            "ingredient": "Irish Whiskey"
+        },
+        {
+            "quantity": "3/4",
+            "measure": "ounces",
+            "ingredient": "coffee liqueur"
+        },
+{
+            "quantity": "3/4",
+            "measure": "ounces",
+            "ingredient": "Irish cream liqueur"
+        },
+        {
+            "quantity": "4",
+            "measure": "ounces",
+            "ingredient": "coffee"
+        },
+        
+    
+    ],
+    "directions": [
+        "Gather the ingredients.",
+        "In a warm mug or Irish coffee glass, pour the Irish whiskey, coffee liqueur, and Irish cream.",
+        "Fill with hot coffee.",
+        "Stir to combine",
+        "Top with whipped cream."
+    ],
+    "image": "IrishWinter.jpg",
```

Can we make this all lowercase and hyphens?
```suggestion
    "image": "irish-winter.jpg",
```

---

Also, remove the old image.

---

i already removed @alfg

---

I still can see `src/recipes/img/IrishWinter.jpg` in this PR>